portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,117 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+  background: #0a192f;
+  font-family: 'Ubuntu', sans-serif;
```

Comment:
Keep these declarations in alphabetical order for consistency and ease of maintenance

Reply:
Oh, didn't know that. Thank you!